packages/circuit-ui/components/FieldAtoms/FieldDescription.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2020, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { HTMLAttributes } from 'react';
+import { css } from '@emotion/react';
+
+import styled from '../../styles/styled';
+import { typography } from '../../styles/style-mixins';
+
+import { CLASS_DISABLED } from './constants';
+
+export interface FieldDescriptionProps extends HTMLAttributes<HTMLSpanElement> {
+  validationHint?: string;
```

Comment:
I have concerns about renaming `validationHint` to `description`. I think this would be confusing for developers without better-defined patterns around form validations.

Reply:
Yeah we probably want to at least think of both so developers, when they migrate, can choose either to pass their legacy `validationHint`s as `description`s, or to opt into whatever form submission pattern we recommend (not sure how much we can build into the web design system, but we can probably define guidelines)